tests/test_utils.py

```
@@ -161,3 +161,37 @@ def print_performance_summary(
     print(f'Average RTF: {rtf_metrics["average_rtf"]:.2f}x (аудио секунд за секунду обработки)')
     print(f'RTF range: {rtf_metrics["minimum_rtf"]:.2f}x - {rtf_metrics["maximum_rtf"]:.2f}x')
     print(f'Processing {"faster" if rtf_metrics["processing_faster_than_realtime"] else "slower"} than real-time')
+
+
+def validate_asr_results(results, expected_count=None, check_timestamps=True):
```

Ruff (0.11.9)

Missing return type annotation for public function `validate_asr_results`

Add return type annotation: `None`

(ANN201)